### PR TITLE
Add data for Periodic Background Sync

### PR DESCRIPTION
**Pull request by jpmedley**
https://www.chromestatus.com/feature/5689383275462656

### CODE REVIEW

File: api/PeriodicSyncEvent.json [hunk 1]
```diff
@@ -34,10 +35,10 @@
             "version_added": false
           },
           "samsunginternet_android": {
-            "version_added": false
+            "version_added": true
```
Comment (ddbeck) [inline api/PeriodicSyncEvent.json:38]: Where did the Samsung Internet data come from? I'm not saying it's wrong, but it doesn't exactly square with the browser version mappings we have in `/browsers/samsung_internet.json`.

The Chrome, WebView, and MDN URLs look fine, so this is the only part I'm concerned about.

Comment (jpmedley) [reply api/PeriodicSyncEvent.json]: This is me forgetting that Samsung doesn't fork the current version of Chrome. I've changed this to false everywhere.

Comment (vinyldarkscratch) [reply api/PeriodicSyncEvent.json]: Looks like a few entries are still left as `true` in PeriodicSyncManager?

Comment (jpmedley) [reply api/PeriodicSyncEvent.json]: Sorry about that. I think I got them this time.

Comment (vinyldarkscratch) [reply api/PeriodicSyncEvent.json]: Not a problem, looks good now!

**Pull request merged**